deploy/prod/values.yaml

```diff
@@ -20,21 +20,21 @@ service:
 
 resources:
   requests:
-    cpu: 500m
+    cpu: 250m
```

Comment:
When we get to testing this in a live environment, with significant load, it will be important to look for "immediate exhaustion" of the containers with this level of cpu.